routers/user/setting/account.go

```diff
@@ -262,6 +262,7 @@ func UpdateUIThemePost(ctx *context.Context, form auth.UpdateThemeForm) {
 		return
 	}
 
+	ctx.SetCookie("themeNow", ctx.User.Theme, nil, setting.AppSubURL, setting.SessionConfig.Domain, setting.SessionConfig.Secure, true)
```

Comment:
Maybe name it `current_theme` which would be more syntactival correct and follow the casing convention of other cookies like `i_like_gitea`.